src/styles/variables.scss

```diff
@@ -0,0 +1,8 @@
+$grey: #eee;
+$pink: #ff69b4;
+$day-size: 100px;
+$border-size: 1px;
+$font-size: 30px;
+$gap-size: 1px;
+$padding-size: 10px;
+$animation-duration: 0.5s;
```

Comment:
good practice is to leave 1 line at the end of the file (please do this fix in each your file)